pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
 python = ">=3.8.0,<4"
 datadog = ">=0.51.0,<1.0.0"
 wrapt = "^1.11.2"
-ddtrace = ">=2.20.0"
+ddtrace = ">=2.20.0,<4"
```

Comment:
Looks like we are covering two major versions here, 2.2 and 3, is this intended?

Reply:
Yes. For now we are supporting both.